packages/neon-nep5/src/main.ts

```diff
@@ -141,3 +141,78 @@ export async function getToken(
     throw err;
   }
 }
+
+/**
+ * Retrieves the complete information about all (or a list of) tokens.

```

Comment:
`all (or a list of)` suggests that the scriptHashArray has two interpretations but I don't see that. I think it will be good to include in the docstring the properties returned.

Reply:
Modified the text as that was unintended :P